ballerina/client_endpoint.bal

```diff
@@ -321,6 +321,47 @@ public isolated client class Client {
         return rename(self, origin, destination);
     }
 
+    # Moves a file from one location to another within
+    # the same FTP server.
+    # ```ballerina
+    # ftp:Error? response = client->move(sourcePath, destinationPath);
+    # ```
+    #
+    # + sourcePath - The source file location
+    # + destinationPath - The destination file location
+    # + return - `()` or else an `ftp:Error` if failed to establish
+    #            the communication with the FTP server
+    remote isolated function move(string sourcePath, string destinationPath) returns Error? {
+        return move(self, sourcePath, destinationPath);
+    }
+
+    # Copies a file from one location to another within
+    # the same FTP server.
+    # ```ballerina
+    # ftp:Error? response = client->copy(sourcePath, destinationPath);
+    # ```
+    #
+    # + sourcePath - The source file location
+    # + destinationPath - The destination file location
+    # + return - `()` or else an `ftp:Error` if failed to establish
+    #            the communication with the FTP server
+    remote isolated function copy(string sourcePath, string destinationPath) returns Error? {
```

Comment:
How do we handle duplicate? Do we replace the existing file or do we return an error?

I think may be we can expose any options like this as an additional argument similar to file API: https://central.ballerina.io/ballerina/file/1.12.0#copy